src/main/java/racingcar/domain/input/PositiveNumber.java

```diff
@@ -0,0 +1,34 @@
+package racingcar.domain.input;
+
+import racingcar.constant.InputExceptionMessage;
+
+public class PositiveNumber {
+
+    protected final int positiveNumber;
```

Comment:
```suggestion
public class RoundCount {

    protected final int roundCount;
```

그럴 바에는 그냥 PositiveNumber 를 RoundCount 라고 이름을 바꾸어서,
PositiveNumber 와 RoundCount 의 역할을 합쳐버리는 것은 어떻게 생각하시나요?

"RoundCount 는 PositiveNumber 이어야 한다." 라는 요구사항에서,
이 요구사항을 RoundCount 를 PositiveNumber 로 구현해야한다고 해석할 수도 있지만,
RoundCount 는 객체 생성시에 PositiveNumber 인지 체크하는 "조건"을 확인해야 한다 라고 해석할 수 있습니다.

즉 요구사항을 구현하기 위해서 필요한게, "멤버 상태" 인지 "메서드 로직" 인지는
개발자 본인이 마음대로 설계할 수 있습니다.

요구사항을 그대로 하려하지 마시고 (보통 요구사항을 건네주시는 기획자분들은 개발을 잘 모릅니다.)
요구사항의 의도가 그 핵심 가치를 파악하려 하면 좋을 것 같습니다.

Reply:
요구사항의 핵심 가치를 파악하라는 좋은 피드백 감사합니다~
좀더 다듬어서 다시 리뷰요청드리겠습니다~~